orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -85,7 +85,8 @@ class DependentPipelineStarter implements ApplicationContextAware {
       parentExecution      : parentPipeline,
       parentPipelineStageId: parentPipelineStageId,
       parameters           : [:],
-      strategy             : suppliedParameters.strategy == true
+      strategy             : suppliedParameters.strategy == true,
+      correlationId        : parentPipeline.id + "_" + pipelineConfig.id + "_" + parentPipeline.startTime
```

Comment:
why `pipelineConfig.id`? would that change from execution to execution?

Reply:
A parent pipeline might have multiple child pipelines that trigger on completion.
Start time accounts for a restart, since we change the start time if the pipeline restarts.